roles/kubernetes/preinstall/tasks/0040-set_facts.yml

```diff
@@ -70,11 +70,21 @@
 
     - name: get currently configured nameservers
       set_fact:
-        configured_nameservers: "{{ resolvconf_slurp.content | b64decode | regex_findall('\\s*nameserver\\s*(.*)') | ipaddr }}"
+        configured_nameservers: "{{ resolvconf_slurp.content | b64decode | regex_findall('^nameserver\\s*(.*)', multiline=True) | ipaddr }}"
```

Comment:
this change is to fix the mis regex for the  " #nameserver " comment .

Reply:
I feel it is fine to go without this change part to fix the original issue only for making this pull request simple.

Reply:
HI @oomichi , It has changed